data/assets/examples/renderable/renderablenodearrow/nodearrow_appearance.asset

```diff
@@ -0,0 +1,40 @@
+-- Custom Appearance (Colored & Inverted)
+-- This example shows an arrow pointing from one scene graph node in the direction of
+-- another. Here, it is created with the Solar System barycenter as start node and
+-- Earth as end node, but the arrow direction is inverted so that it points towards
+-- the Solar System barycenter. Some appearance related properties are also changes to
+-- customize the look of the arrow, but default values are used for its size.
+
+local earth = asset.require("scene/solarsystem/planets/earth/earth")
+local sunTransforms = asset.require("scene/solarsystem/sun/transforms")
+
+local Node = {
+  Identifier = "RenderableNodeArrow_Example_Appearance",
+  -- Parent to the start node, so that when we focus on the arrow this is where we end up
```

Comment:
Since there is no parent in this node, should we remove this comment?

Reply:
Ah.. Yes, or bring back the parent node :)